packages/flutter_tools/lib/src/localizations/gen_l10n_types.dart

```diff
@@ -318,6 +317,23 @@ class Placeholder {
   }
 }
 
+@immutable
+class Namespace {
```

Comment:
This class doesn't seem useful, why not use String as type for namespace directly